.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,87 @@
+name: Release PR
+on:
+  repository_dispatch:
+    types: [release-published]
+
+jobs:
+  release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Cache 📦
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Setup Node Environment ⬢
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Update version in package.json
+        id: version-bump
+        uses: jaywcjlove/github-action-package@v1.3.0
+        with:
+          path: "./package.json"
+          version: ${{ github.event.client_payload.release_version }}
+
+      - run: echo "version - ${{ steps.info.outputs.version }} - ${{ github.event.client_payload.release_version }}"
+
+      - name: Install 🔧
+        run: npm install
+
+      - name: Build 🔧
+        run: npm run build
+
+      - name: Commit files from the build
+        run: |
+          git config --local user.name  ${{ github.actor }}
+          git add --all
+          git commit -m "Release PR commit for ${{ github.event.client_payload.release_version }}"
+
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: release-pr-${{ github.event.client_payload.release_version }}
+
+      - name: Tag release
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ github.event.client_payload.release_version }}
+          message: "Validator Release ${{ github.event.client_payload.release_version }}"
+
+      - uses: actions/github-script@v6
+        id: create-release-pr
+        name: Create Release PR
+        with:
+          # TODO: I need help setting up permissions for this
+          github-token: ${{secrets.TEXTILEIO_MACHINE_ACCESS_TOKEN}}
```

Comment:
Not sure how this secret gets added.  cc: @asutula

Reply:
We have it as an org wide secret. So if we have already added it and named it that, you can access it here.